src/api/log.js

```diff
@@ -13,9 +13,12 @@ import { join } from '../utils/join.js'
  * @param {FsClient} args.fs - a file system client
  * @param {string} [args.dir] - The [working tree](dir-vs-gitdir.md) directory path
  * @param {string} [args.gitdir=join(dir,'.git')] - [required] The [git directory](dir-vs-gitdir.md) path
+ * @param {string=} args.filepath optional get the commit for the filepath only
```

Comment:
We decided that there should be single parameter `string | string[]`

Reply:
multi filepaths? maybe should add another `log-files` api to keep the returned result unchanged.

Reply:
Or do your mean to insert the all commits of filepaths to the result? Manually indicate which is the change history of the same file with different file names.

Reply:
Sorry my mistake, I'm reading too many issues at once. That change with filepath was for git.add in #1228